README.md

```diff
@@ -38,48 +38,57 @@ npm start
 
 Learn more for building your first Wechaty bot at <https://github.com/wechaty/wechaty-getting-started>
 
-## 项目介绍
+- ## 项目介绍
 
-“开源软件供应链点亮计划-暑期2020”（以下简称暑期2020）是由中科院软件所与 openEuler 社区共同举办的一项面向高校学生的暑期活动。旨在鼓励在校学生积极参与开源软件的开发维护，促进国内优秀开源软件社区的蓬勃发展。
```

Comment:
In fact, this repo is supported by **both** Summer-2020 and Summer-2021, you'd better show it.